src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -2086,10 +2079,11 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
             //
 
             // Adding the value of $(MSBuildExtensionsPath*) property to the list of search paths
-            var prop = _data.GetProperty(refKindInProject.StringRepresentation);
+            var prop = _data.GetProperty(fallbackSearchPathMatch.PropertyName);
+            var fallbackExtensionPaths = fallbackSearchPathMatch.SearchPaths.ToList();
```

Comment:
It was previously copied to a new list elsewhere so it could be modified here (at least that's what I assumed). I fixed that up a little bit in the latest iteration and added your suggestion, which I like much better.